test/signer.ts

```diff
@@ -573,6 +573,52 @@ describe('signer', () => {
         );
       });
 
+      it('should not throw with expiration of exactly 7 days', async () => {
+        const file = new Storage({
+          projectId: 'xxxx',
+          credentials: {
+            type: 'service_account',
+            private_key: crypto.generateKeyPairSync('rsa', {
+              modulusLength: 512,
+              publicKeyEncoding: {
+                type: 'spki',
+                format: 'pem',
+              },
+              privateKeyEncoding: {
+                type: 'pkcs8',
+                format: 'pem',
+              },
+            }).privateKey,
+            client_email: 'xxxx',
+            client_id: 'xxx',
+          },
+        })
+          .bucket(BUCKET_NAME)
+          .file(FILE_NAME);
+        const ACCESSIBLE_AT = NOW.valueOf();
+        const SEVEN_DAYS_IN_SECONDS = 7 * 24 * 60 * 60;
+        const SEVEN_DAYS_IN_MS = SEVEN_DAYS_IN_SECONDS * 1000;
+        await assert.doesNotReject(
+          async () => {
+            await file.getSignedUrl({
```

Comment:
I think this can be accomplished without the need of pulling in `file`. For example: https://github.com/googleapis/nodejs-storage/blob/2c967f6bee97f0f1a271de23c380c545dde532fc/test/signer.ts#L208

Reply:
Yes agreed. Please see latest commit.